packages/adapters/src/pipedream-connector.ts

```
@@ -239,17 +274,57 @@ export class PipedreamConnector implements ManagedConnectorProvider {
   }
 
   async revoke(externalId: string, context: AdapterContext): Promise<void> {
-    const accounts = await this.accounts(context, externalId);
+    let accounts;
```

Biome (2.5.8)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)